e2e/testcases/helm_sync_test.go

```diff
@@ -730,6 +733,7 @@ func TestHelmARTokenAuth(t *testing.T) {
 		nomostesting.SyncSource,
 		ntopts.Unstructured,
 		ntopts.RequireGKE(t),
+		ntopts.RequireHelmProvider,
 		ntopts.RequireHelmArtifactRegistry(t),
```

Comment:
Could have `RequireHelmArtifactRegistry` imply `RequireHelmProvider` as well, although that might require a bit more refactoring

Reply:
Done